### PR TITLE
Update actions.py

### PR DESCRIPTION
**Pull request by Matthew-Imaginary**
updated user_message

### CODE REVIEW

File: src/cogs/helpers/actions.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ async def challenge_selection(self):
 
         await self.ticket_channel.set_permissions(member, read_messages=True,
                                                   send_messages=None)
-        user_message = await self.ticket_channel.send("What have your tried so far?")
+        user_message = await self.ticket_channel.send("Please explain what you've tried so far and what you need help with")
```
Comment (radatta) [inline src/cogs/helpers/actions.py:271]: Looks a bit weird without punctuation imo

Comment (Matthew-Imaginary) [reply src/cogs/helpers/actions.py:271]: you can add some

Comment (radatta) [reply src/cogs/helpers/actions.py:271]: Can't think of much rn.

**Comment by radatta** [comment]
Thanks for the PR!

**Pull request merged**